es-vue-base/src/lib-components/EsAccordion.vue

```diff
@@ -1,18 +1,24 @@
 <template>
     <div class="EsAccordion border-bottom border-light rounded-bottom">
-        <header role="tab">
-            <es-button
-                class="EsAccordion-button h-auto align-items-center d-flex font-weight-bold justify-content-between px-100 px-sm-200 py-100 rounded-0 text-body text-left text-decoration-none w-100"
+        <header
+            role="tab"
+            class="position-relative d-inline-block w-100">
+            <component
+                :is="headingTag"
+                class="EsAccordion-heading mb-0 h-auto align-items-center d-flex font-weight-bold justify-content-between px-100 px-sm-200 py-100 rounded-0 text-body text-left text-decoration-none"
```

Comment:
A couple things here:
- Some of these utility classes were only here to override certain EsButton styles (at least `h-auto`, `font-weight-bold`, `rounded-0`, `text-left`, `text-decoration-none`, I believe), and we may no longer need them. Can we remove the unnecessary ones?
- Changing the default to an `<h3>` is appropriate, but it does dramatically change the rendered text size on desktop. [CED-799](https://energysage.atlassian.net/browse/CED-799) will solve for the ability to pass in a `variant` prop to customize the font size, among other styling, but in order to minimize breaking visual changes in this PR (and set up what will be needed for the `'default'` variant, let's set `font-size-100` here to keep it the same size it was before.